src/Events/Calendar_Embeds/Render.php

```diff
@@ -0,0 +1,1306 @@
+<?php
+/**
+ * Render External Embed Calendars.
+ *
+ * @since TBD
+ * @package TEC/Events/Calendar_Embeds
+ */
+
+namespace TEC\Events\Calendar_Embeds;
+
+use Tribe\Utils\Taxonomy;
+use Tribe\Events\Views\V2\Manager as Views_Manager;
+use Tribe\Events\Views\V2\Theme_Compatibility;
+use Tribe\Events\Views\V2\View;
+use Tribe\Events\Views\V2\View_Interface;
+use Tribe\Events\Views\V2\Views\Month_View;
+use Tribe\Utils\Element_Classes;
+use Tribe__Context as Context;
+use Tribe__Events__Main as TEC;
+use Tribe__Utils__Array as Arr;
+use Tribe\Events\Views\V2\Views\Day_View;
+use Tribe__Repository__Interface as Repository_Interface;
+
+/**
+ * Class for rendering the External Calendar Embeds.
+ *
+ * @since TBD
+ *
+ * @package TEC/Events/Calendar_Embeds
+ */
+class Render {
+	/**
+	 * Prefix for the transient where we will save the base values for the
+	 * setup of the context of the embed.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	const TRANSIENT_PREFIX = 'tec_events_shortcode_tribe_events_params_';
```

Comment:
There is a lot of "shortcode" in here, and a huge amount of duplication brought over from ECP.

Reply:
Just pick a term and stick to it - let's not bounce back and forth between the two

Reply:
Kinda picked Views but also used embed when it made more sense